.build/tasks/New-Release.GitHub.build.ps1

```diff
@@ -109,7 +109,10 @@ task Publish_release_to_GitHub -if ($GitHubToken -and (Get-Module -Name PowerShe
             ErrorAction    = 'Stop'
         }
 
-        Write-Build DarkGray "Checking if the Release exists: `r`n Get-GithubRelease $($getGHReleaseParams | Out-String)"
+        $displayGHReleaseParams = $getGHReleaseParams.Clone()
+        $displayGHReleaseParams.Remove('AccessToken')
```

Comment:
What do you think of just change the value to still show that it is part of the call?

```suggestion
        $displayGHReleaseParams.['AccessToken'] = 'Redacted'
```